### PR TITLE
feat(dashboard): display error message on user creation

### PR DESCRIPTION
**Pull request by IGassmann**
## Description

This displays the reason why a user couldn't get created on the "Team Management" page:

Inngest Cloud

## Type of change (choose one)
- [ ] Chore (refactors, upgrades, etc.)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [ ] I've linked any associated issues to this PR.
- [ ] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export function UserCreator() {
 
     const res = await createUser({ input });
     if (res.error) {
-      toast.error('Failed to create user');
+      toast.error(`Failed to create user: ${res.error.graphQLErrors?.[0]?.message}`);
```
Comment (djfarrelly) [inline ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx:70]: Can we append something like "Please contact support" and if possible in Sonner, link to `https://app.inngest.com/support`? we can also let it have longer duration as well so someone has time to click on the link.

Comment (IGassmann) [reply ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx]: Done :) https://github.com/inngest/inngest/pull/830/commits/d9375c334e4a563c8e5d90729d7448a3f174ff2e

File: ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx [hunk 2]
```diff
@@ -67,7 +67,14 @@ export function UserCreator() {
 
     const res = await createUser({ input });
     if (res.error) {
-      toast.error('Failed to create user');
+      toast.error('Failed to create user', {
+        description: res.error.graphQLErrors?.[0]?.message,
+        action: {
+          label: 'Contact Support',
```
Comment (goodoldneon) [inline ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx:73]: Does this create a button? I don't see it in the screenshot

Comment (IGassmann) [reply ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx:73]: Yep, I didn't update the screenshot:

Arc 2023-11-10 at 15 55 22@2x

**Review by goodoldneon — APPROVED** [review]

**Pull request merged**